Systems/glass-effects.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<PropertyList>
+
+    <filter>
+        <name>splashX</name>
+        <type>gain</type>
+        <input>
+            <property>/fdm/jsbsim/glass-effects/splashx</property>
+        </input>
+        <output>
+            <property>/environment/aircraft-effects/splash-vector-x</property>
```

Comment:
Have you tried adding `<output>/environment/aircraft-effects/splash-vector-x</output>` to within `<fcs_function>`? (At the same level as `<function>`)